newscoop/library/Newscoop/Services/Plugins/ManagerService.php

```diff
@@ -98,7 +109,7 @@ public function installPlugin($pluginName, $version, $output, $notify = true)
             throw new \Exception("Plugin name is invalid, try \"vendor/plugin-name\"", 1);
         }
 
-        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar require --no-update ' . $pluginName .':' . $version .' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar update ' . $pluginName .'  --prefer-dist --no-dev -n');
+        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.qphar require ' . $pluginName .':' . $version);
```

Comment:
Yeah, if you notice, require was being called with --no-update, before
calling update. I couldn't see any need for this.

On Mon, Mar 21, 2016 at 9:57 PM, Rafał Muszyński <notifications@github.com>
wrote:

> In newscoop/library/Newscoop/Services/Plugins/ManagerService.php
> <https://github.com/sourcefabric/Newscoop/pull/1347#discussion_r56895711>:
>
> > @@ -98,7 +109,7 @@ public function installPlugin($pluginName, $version, $output, $notify = true)
> >              throw new \Exception("Plugin name is invalid, try \"vendor/plugin-name\"", 1);
> >          }
> >
> > -        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar require --no-update ' . $pluginName .':' . $version .' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar update ' . $pluginName .'  --prefer-dist --no-dev -n');
> > +        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.qphar require ' . $pluginName .':' . $version);
>
> yeah it will also download the package
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcefabric/Newscoop/pull/1347/files/092aafa5c9f683790542be69c37d806eafec3544#r56895711>
>